src/generate/generate.nim

```diff
@@ -0,0 +1,47 @@
+import std/[os, strformat, strscans, strutils, terminal]
+import ".."/[cli, helpers]
+
+proc writeError(description, path: string) =
```

Comment:
This feels like something that should go in the `helpers` module possibly?

Reply:
Indeed, it's been missing in some other places too. I've opened https://github.com/exercism/configlet/issues/325 to track it.